### PR TITLE
Time picker localization in opening hours editor

### PR DESCRIPTION
**Pull request by RicoElectrico**
For the time being, Polish and English are localized.

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -9843,6 +9843,17 @@
     zh-Hans = 下一页
     zh-Hant = 下一頁
 
+  [editor_time_from]
+    comment = Tab title in the time picker
+    tags = android
+    en = From
```
Comment (biodranik) [inline data/strings/strings.txt:9849]: Thanks! Can you please extract the correct From and To translations from these strings, generated by [this script](https://github.com/organicmaps/organicmaps/pull/5118/files#diff-f37cf5a5a9e99414057fcfa717fa9b85caf349abcfa3ebce55b91c25061d5e8a)?

```
en = From 6:00 to 11:00
ar = من الساعة 6:00 إلى 11:00
be = З 6.00 да 11.00
bg = От 6:00 до 11:00 ч.
ca = De 6:00 a 11:00
cs = Od 6:00 do 11:00
da = Fra 6:00 til 11:00
de = Von 6:00 bis 11:00
el = Από τις 6:00 έως τις 11:00
en-GB = From 6:00 to 11:00
es = De 6:00 a 11:00
et = Kell 6:00 kuni 11:00
eu = 6:00etatik 11:00etara
fa = از ساعت 6:00 الی 11:00
fi = Klo 6:00-11:00
fr = De 6h00 à 11h00
he = משעה 6:00 עד 11:00
hu = 6:00-tól 11:00-ig
id = Dari pukul 6:00 hingga 11:00
it = Dalle 6:00 alle 11:00
ja = 6:00から11:00まで
ko = 6:00~11:00
mr = 6:00 ते 11:00 पर्यंत
nb = Fra 6:00 til 11:00
nl = Van 6:00 tot 11:00
pl = Od 6:00 do 11:00
pt = Das 6:00 às 11:00
pt-BR = Das 6:00 às 11:00
ro = De la 6:00 la 11:00
ru = С 6:00 до 11:00
sk = Od 6:00 do 11:00
sv = Från 6:00 till 11:00
th = เวลา 6.00 - 11.00 น
tr = 6:00 - 11:00 arası
uk = З 6:00 до 11:00
vi = Từ 6:00 đến 11:00
zh-Hans = 从6:00到11:00
zh-Hant = 從 6:00 到 11:00
```

File: data/strings/strings.txt [hunk 2]
```diff
@@ -9847,12 +9847,82 @@
...
+    da = Fra
+    de = Von
+    el = Από τις
+    es = De
+    et = Kell
+    eu = etatik
+    fa = از ساعت
+    fi = alkaen
+    fr = De
+    he = משעה 6:00 עד 11:00
```
Comment (biodranik) [inline data/strings/strings.txt:9864]:
```suggestion
    he = משעה
```

File: data/strings/strings.txt [hunk 3]
```diff
@@ -9847,12 +9847,82 @@
...
+    id = Dari pukul
+    it = Dalle
+    ja = から
+    ko = 부터
+    mr = ते
+    nb = Fra
+    nl = Van
     pl = Od
+    pt = Das
+    pt-BR = Das
```
Comment (biodranik) [inline data/strings/strings.txt:9875]:
```suggestion
```

Comment (biodranik) [reply data/strings/strings.txt:9875]: Duplicate sub-regional languages are not needed if they are the same with base ones (pt).

File: data/strings/strings.txt [hunk 4]
```diff
@@ -9847,12 +9847,82 @@
...
+    uk = З
+    vi = Từ
+    zh-Hans = 从
+    zh-Hant = 從
 
   [editor_time_to]
     tags = android
-    en = To
-    pl = Do
+    en = to
```
Comment (biodranik) [inline data/strings/strings.txt:9889]: Does lower cased `to` look better in that context? If capital one is better, then should the first letters in other langs be converted to capital ones?

File: data/strings/strings.txt [hunk 5]
```diff
@@ -9847,12 +9847,82 @@
...
+    id = hingga
+    it = alle
+    ja = まで
+    ko = 까지
+    mr = पर्यंत
+    nb = til
+    nl = tot
+    pl = do
+    pt = às
+    pt-BR = às
```
Comment (biodranik) [inline data/strings/strings.txt:9915]:
```suggestion
```

File: data/strings/strings.txt (hunk 4 above)
Comment (RicoElectrico) [reply data/strings/strings.txt:9889]: It should not matter as the picker converts them to all caps.
Do you know any tool that can convert a string to "sentence case" and do so reliably for all scripts?

Comment (biodranik) [reply data/strings/strings.txt:9889]: Do all caps look good?

You can select the first LTR word in Visual Studio Code and select Title Case. I would not touch RTL langs and non-latin/non-cyrillic scripts.

Comment (RicoElectrico) [reply data/strings/strings.txt:9889]: The all caps were already there before all my changes. It's a Material Design convention I think.
I managed to convert strings to sentence case.

File: data/strings/strings.txt (hunk 5 above)
Comment (RicoElectrico) [reply data/strings/strings.txt:9915]: Fixed.

File: data/strings/strings.txt (hunk 3 above)
Comment (RicoElectrico) [reply data/strings/strings.txt:9875]: Fixed.

File: data/strings/strings.txt (hunk 2 above)
Comment (RicoElectrico) [reply data/strings/strings.txt:9864]: Oops! :) Fixed.

File: data/strings/strings.txt [hunk 6]
```diff
@@ -9843,6 +9843,85 @@
     zh-Hans = 下一页
     zh-Hant = 下一頁
 
+  [editor_time_from]
+    comment = Tab title in the time picker
```
Comment (biodranik) [inline data/strings/strings.txt:9847]:
```suggestion
    comment = Tab title in the Edit Opening Hours time picker
```

Comment (biodranik) [reply data/strings/strings.txt:9847]: Please add a similar comment to the To field. It will help translators with the context.

**Review by biodranik — APPROVED** [review]

File: data/strings/strings.txt (hunk 6 above)
Comment (RicoElectrico) [reply data/strings/strings.txt:9847]: Done.

**Review by biodranik — APPROVED** [review]
Thank you very much!

**Comment by biodranik** [comment]
The last step, for a clean history: please squash all commits except regenerated into one, and all regenerated commits into another one.

It helps to avoid merge conflicts.

**Comment by RicoElectrico** [comment]
> The last step, for a clean history: please squash all commits except regenerated into one, and all regenerated commits into another one.
> 
> It helps to avoid merge conflicts.

Done!

**Comment by biodranik** [comment]
Awesome! Thank you for your efforts, we hope you learned something new ;-)

**Pull request merged**